x-pack/plugins/ml/server/routes/saved_objects.ts

```diff
@@ -59,10 +59,10 @@ export function savedObjectsRoutes(
    *
    * @api {get} /api/ml/saved_objects/sync Sync job saved objects
    * @apiName SyncJobSavedObjects
-   * @apiDescription Create saved objects for jobs which are missing them.
-   *                 Delete saved objects for jobs which no longer exist.
-   *                 Update missing datafeed ids in saved objects for datafeeds which exist.
-   *                 Remove datafeed ids for datafeeds which no longer exist.
+   * @apiDescription Synchronizes saved objects for jobs. Saved objects will be created for jobs which are missing them,
+   *                 and saved objects will be deleted for jobs which no longer exist.
+   *                 Updated missing datafeed IDs in saved objects for datafeeds which exist, and
```

Comment:
```suggestion
   *                 Updates missing datafeed IDs in saved objects for datafeeds which exist, and
```

Reply:
Done in https://github.com/elastic/kibana/pull/111298/commits/2148278afae5c49dbed7850e9de6678792cb0434